src/components/ui/Chat/ChatBubble/index.tsx

```diff
@@ -12,10 +12,12 @@ export interface ChatBubbleProps extends Omit<HTMLAttributes<HTMLDivElement>, 'c
   variant: MessageVariant;
   /** The name of the user that sent the message. */
   senderName: string;
-  /** The text content of the message. */
-  content: string;
+  /** The timestamp of the message being sent. */
+  timestamp?: string;
   /** Determines if the name should be shown or not. */
   showName?: boolean;
```

Comment:
I know you didn't write this code, but like below, I think we can get rid of this prop too.

Reply:
Do you mean we only need `timestamp`?

Reply:
Yeah, just like with the name, I think instead of passing `showTimestamp={false}` you could just not pass a `timestamp` prop.